src/librustc_save_analysis/dump_visitor.rs

```diff
@@ -1233,10 +1233,13 @@ impl<'l, 'tcx: 'l, 'll, O: DumpOutput + 'll> DumpVisitor<'l, 'tcx, 'll, O> {
     fn process_use_tree(&mut self,
                          use_tree: &'l ast::UseTree,
                          id: NodeId,
-                         parent_item: &'l ast::Item,
+                         root_item: &'l ast::Item,
```

Comment:
Could you add a comment to the function here explaining what the `root_item` is please? Also perhaps what a `use_tree` is.